Onboarding/QA/profile-pregnant-nurse.md

```diff
@@ -4,33 +4,7 @@
 trigger(on: "MESSAGE RECEIVED") when has_only_phrase(event.message.text.body, "nurse")
 
 ```
-# Onboarding: Profile Pregnant Nurse
```

Comment:
Is there a reason we got rid of this documentation?

Reply:
Probably a finger slip and deleted the block :-/